modules/settings/assets/js/components/color-picker/style.css

```diff
@@ -0,0 +1,29 @@
+.custom-color-picker.react-colorful {
```

Comment:
```suggestion
.widget-settings-color-picker.react-colorful {
```

let give it a more specific class then just "custom-color-picker"